### PR TITLE
Add glossary alts for plural terms

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
I think this is good to go, @tiborsimko!

### CODE REVIEW

File: invenio_opendata/base/templates/helpers/text/glossary.json [hunk 1]
```diff
@@ -1,405 +0,0 @@
-[
```
Comment (tiborsimko) [inline invenio_opendata/base/templates/helpers/text/glossary.json]: WRT the removal of the old glossary file, it is still used by https://github.com/cernopendata/opendata.cern.ch/blob/master/invenio_opendata/base/views.py#L566

Comment (RaoOfPhysics) [inline invenio_opendata/base/templates/helpers/text/glossary.json]: Hmm. So, what should I do at the moment?

Comment (tiborsimko) [inline invenio_opendata/base/templates/helpers/text/glossary.json]: Nothing on your side, me or @pamfilos shall look at fixing this while merging your PR.

**Comment by RaoOfPhysics** [comment]
@katilp: Please check the definitions for the new terms added.

**Comment by katilp** [comment]
@RaoOfPhysics These are not yet visible in http://opendatadev.cern.ch/glossary, will they appear there?

**Comment by RaoOfPhysics** [comment]
@katilp: I believe the PR needs to be merged before it'll appear there. For now, can you check the JSON file? It's not pretty, but…

**Comment by tiborsimko** [comment]
Thanks, squashed and merged as cb0b467b6e130f346949f4ae2473be4436144b2c and c023406291d118a37da45aad751f5c290eca2208, and quick-fixed escaping in 44d40d14e4f334b16e9a638ac116df04e518091c.

**Pull request closed without merging**